llvm/test/CodeGen/AMDGPU/gcn-reg-pressure-true16-integer-overflow.mir

```diff
@@ -0,0 +1,70 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -x mir -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1102 -run-pass=machine-scheduler %s -o - | FileCheck %s
+
+--- |
```

Comment:
Probably don't need IR section, and can compact register numbers with -run-pass=none